extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -50,6 +51,13 @@
     @ConfigItem
     Optional<String> clientId;
 
+    /**
+     * The time the adapter can wait for before connecting to the OIDC server.
```

Comment:
```suggestion
     * The time the adapter can wait for before connecting to the OIDC server.
```
For now it seems to be the time the adapter **will** wait. Not sure if we want that or if we want to retry from time to time?

Reply:
@gsmet I've updated the code to retry every 2 seconds for up to the` connectionDelay` time in seconds, by default or if this property is set to less than 2 secs it is just a single connection attempt